### ISSUE TITLE
Plugin Not loading

### ISSUE THREAD

**Issue by oyinkan21** [DBraun/ChucKDesigner#20]
Hi. I just downloaded ChucKDesigner and wanted to fiddle around with it a bit however I have an issue in Chuck Audio where it says it failed to load the plugin. How do I fix this? 
![cerror](https://github.com/DBraun/ChucKDesigner/assets/161485609/95eca980-56fa-423a-9784-fdccb0e0d75b)
![cerror](https://github.com/DBraun/ChucKDesigner/assets/161485609/af174776-7b40-4456-aee9-c8f93fd80fc1)

**Comment by DBraun** [comment]
What’s your OS and TouchDesigner version? Can you try creating and saving a new Toe project next to ChucKDesigner.toe, then reopening the project and trying to use the plugins? I have sometimes observed issues in which the Plugins need to be deleted and recreated in the same project when there’s a new version.

**Comment by oyinkan21** [comment]
I tried creating a new Toe project next to ChuckDesigner.toe but it still didn't work. I'm not sure if I just didn't use the plugins correctly. How would I delete and recreate them? Is there a way to work around this?

**Comment by DBraun** [comment]
Assuming you're on Windows and using TouchDesigner 2023.11600, download the Python 3.11 zip [here](https://github.com/DBraun/ChucKDesigner/releases). Unzip the content and put it in the Plugins folder next to ChucKDesigner.toe. Then re-open the project, and it all should work. I have tested it again just now using the the files from GitHub.

**Comment by oyinkan21** [comment]
I just tried doing that but I'm still running into the same issue. Could this be because I'm on Mac and not Windows?

**Comment by DBraun** [comment]
Yes that is why I asked for your operating system earlier. For macOS you need to build ChucKDesigner locally. The instructions are on the homepage: https://github.com/DBraun/ChucKDesigner?tab=readme-ov-file#macos

**Comment by oyinkan21** [comment]
I did build it locally but I'm going to try deleting my cloned repo and redoing it.

**Comment by DBraun** [comment]
Is the issue resolved? The latest [release](https://github.com/DBraun/ChucKDesigner/releases) has precompiled plugins for macOS Apple Silicon (arm64) and Intel (x86_64).
